modules/configuring-a-floating-action-button/proc-configuring-floating-action-button-as-a-dynamic-plugin.adoc

```diff
@@ -16,51 +16,127 @@ To configure a floating action button as a dynamic plugin, complete any of the f
 .Example of a bulk-import plugin as a floating action button
 [source,yaml]
 ----
-  dynamicPlugins:
-    frontend:
-      red-hat-developer-hub.backstage-plugin-bulk-import:
-        # Start of the floating action button configuration
-        mountPoints:
-          - mountPoint: global.floatingactionbutton/config
-            importName: BulkImportPage <1>
-            config:
-              slot: 'page-end'
-              icon: bulkImportIcon
-              label: 'Bulk import'
-              toolTip: 'Register multiple repositories in bulk'
-              to: /bulk-import/repositories
-        # End of the floating action button configuration
-        appIcons:
-          - name: bulkImportIcon
-            importName: BulkImportIcon
-        dynamicRoutes:
-          - path: /bulk-import/repositories
-            importName: BulkImportPage
-            menuItem:
-              icon: bulkImportIcon
-              text: Bulk import
+- package: ./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-bulk-import
+  disabled: false
+  pluginConfig:  
+    dynamicPlugins:
+      frontend:
+        red-hat-developer-hub.backstage-plugin-bulk-import:
+          # Start of the floating action button configuration
+          mountPoints:
+            - mountPoint: global.floatingactionbutton/config
+              importName: BulkImportPage # <1>
+              config:
+                slot: 'page-end'
+                icon: <svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 24 24" height="24px" viewBox="0 0 24 24" width="24px" fill="#e8eaed"><g><rect fill="none" height="24" width="24"/></g><g><path d="M11,7L9.6,8.4l2.6,2.6H2v2h10.2l-2.6,2.6L11,17l5-5L11,7z M20,19h-8v2h8c1.1,0,2-0.9,2-2V5c0-1.1-0.9-2-2-2h-8v2h8V19z"/></g></svg> # <2>
+                label: 'Bulk import'
+                toolTip: 'Register multiple repositories in bulk'
+                to: /bulk-import/repositories
+          # End of the floating action button configuration
+          appIcons:
+            - name: bulkImportIcon
+              importName: BulkImportIcon
+          dynamicRoutes:
+            - path: /bulk-import/repositories
+              importName: BulkImportPage
+              menuItem:
+                icon: bulkImportIcon
+                text: Bulk import
 ----
 <1> (Required) The import name with an associated component to the mount point.
+<2> Use this `svg` value to display a black BulkImportPage icon.
```

Comment:
Would it be better to say that the icon allows
`React.ReactElement`, `SVG image icon` and an`HTML image icon`

can you also mention in the button parameters below ?

Reply:
Okay, should be fine then